src/style.css

```
@@ -1 +1,136 @@
-/* add styles here */
+@font-face {
+    font-family: Avenir;
+    font-weight: 300;
+    src: url("./fonts/Avenir-Book.ttf");
+}
```

Fix the spaces everywhere for better readability
```suggestion
@font-face {
  font-family: Avenir;
  font-weight: 300;
  src: url("./fonts/Avenir-Book.ttf");
}
```